src/AzureFunctionPackage/Functions/Common/config.xml

```diff
@@ -7,14 +7,15 @@
     <merge from="release/dev16.1" to="release/dev16.2" />
     <merge from="release/dev16.2" to="release/dev16.3-preview1" />
     <merge from="release/dev16.3-preview1" to="master" />
+    <merge from="master" to="release/dev16.4-preview1" />
 
     <!-- Roslyn branches (between vs-deps branches) -->
     <merge from="dev15.0.x-vs-deps" to="dev15.9.x-vs-deps" />
     <merge from="dev15.9.x-vs-deps" to="dev16.0-vs-deps" />
     <merge from="dev16.0-vs-deps" to="release/dev16.1-vs-deps" />
     <merge from="release/dev16.1-vs-deps" to="release/dev16.2-vs-deps" />
     <merge from="release/dev16.2-vs-deps" to="release/dev16.3-preview1-vs-deps" />
```

Comment:
should this be `<merge from="release/dev16.2-vs-deps" to="release/dev16.3-preview2-vs-deps" />`?

Reply:
good call, I missed this one